README.MD

```diff
@@ -107,6 +107,20 @@ After setup, it is good to check logs in all related modules for errors.
     * "_inventory.kafka.MarcBibUpdateConsumerVerticle.instancesNumber_": 3
     * "_inventory.kafka.MarcBibUpdateConsumer.loadLimit_": 5
     * "_inventory.kafka.MarcBibUpdateConsumer.maxDistributionNumber_": 100
+* Variables for setting number of partitions of topics:
+  * DI_INVENTORY_INSTANCE_CREATED_READY_FOR_POST_PROCESSING
```

Comment:
DI_INVENTORY_INSTANCE_CREATED_READY_FOR_POST_PROCESSING is no longer in use. What about HOLDINGS_NOT_MATCHED and ITEM_NOT_MATCHED?